modules/gitops-release-notes-1-14-0.adoc

```diff
@@ -0,0 +1,106 @@
+//
+// * release_notes/gitops-release-notes-1-14-0.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-14-0_{context}"]
+= Release notes for {gitops-title} 1.14.0
+
+{gitops-title} 1.14.0 is now available on {OCP} 4.12, 4.13, 4.14, 4.15, and 4.16.
+
+[id="errata-updates-1-14.0_{context}"]
+== Errata updates
+
+[id="RHSA-XXXX:NNNN-gitops-1-14-0-security-update-advisory_{context}"]
+=== RHSA-XXXX:NNNN - {gitops-title} 1.14.0 security update advisory
+
+Issued: 2024-09-18
+
+The list of security fixes that are included in this release are documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-XXXX:NNNN[RHSA-XXXX:NNNN]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="new-features-1-14-0_{context}"]
+== New features
+
+* With this update, users can customize permissions for the Application Controller component in a cluster-scoped Argo CD instance. The {gitops-title} Operator establishes an aggregated `ClusterRole`, and users can configure its permissions by following the standards of an aggregated `ClusterRole`. link:https://issues.redhat.com/browse/GITOPS-4681[GITOPS-4681]
+
+* With this update, the following new features have been introduced for multi-sourced application:
+** Users can access revision history and rollback to a specific version of a multi-source application. Specifically, the *Details* view panel displays the most recent revision or commit details for each source in the multi-source application. Also, the *History* and *Rollback* panel displays the revision history associated with each source, enabling users to select the desired version for rollback. link:https://issues.redhat.com/browse/GITOPS-4647[GITOPS-4647]
+** A new *SOURCES* tab is introduced for viewing and editing the source parameters of a multi-source application. The *SOURCES* tab displays all source parameters associated with the multi-source application, allowing you to edit the parameter values by using a form in the Web UI. This functionality consists of the capabilities for single-source applications, which are now extended to accommodate multiple sources. link:https://issues.redhat.com/browse/GITOPS-4192[GITOPS-4192]
+
+* With this update, Repo server storage used for storing manifests can be extended by mounting Persistent Volumes. The default storage volume utilized by the Repo server is `EmptyDir`, which has limitations and might prove inadequate in scenarios involving numerous repositories, sizable repositories, or repositories containing large manifests. Additional storage can be mounted when required by using the `.spec.repo.volumes` and `.spec.repo.volumeMounts` fields in the Argo CD CR. link:https://issues.redhat.com/browse/GITOPS-4646[GITOPS-4646]
```

Comment:
Seeing some inconsistent capitalization for "Repo server" vs "Repo Server"

```suggestion
* With this update, Repo server storage used for storing manifests can be extended by mounting persistent volumes. The default storage volume utilized by the Repo server is `EmptyDir`, which has limitations and might prove inadequate in scenarios involving numerous repositories, sizable repositories, or repositories containing large manifests. Additional storage can be mounted when required by using the `.spec.repo.volumes` and `.spec.repo.volumeMounts` fields in the Argo CD CR. link:https://issues.redhat.com/browse/GITOPS-4646[GITOPS-4646]
```

Reply:
Fixed.